models/stg_reddit_ads__account_report.sql

```diff
@@ -45,4 +44,5 @@ final as (
     from fields
 )
 
-select * from final
+select * 
+from final
```

Comment:
Small whitespace edit suggestion.

Please apply the same changes to the other staging models.

Reply:
updated